ow_power_system/src/PowerSystemNode.cpp

```diff
@@ -195,9 +194,9 @@ bool PowerSystemNode::initTopics()
   // Construct the PowerSystemNode publishers
   m_mechanical_power_raw_pub = m_nh.advertise<Float64>("mechanical_power/raw", 1);
   m_mechanical_power_avg_pub = m_nh.advertise<Float64>("mechanical_power/average", 1);
-  m_state_of_charge_pub = m_nh.advertise<Float64>("power_system_node/state_of_charge", 1);
-  m_remaining_useful_life_pub = m_nh.advertise<Int16>("power_system_node/remaining_useful_life", 1);
-  m_battery_temperature_pub = m_nh.advertise<Float64>("power_system_node/battery_temperature", 1);
+  m_state_of_charge_pub = m_nh.advertise<owl_msgs::StateOfCharge>("/state_of_charge", 1);
+  m_remaining_useful_life_pub = m_nh.advertise<owl_msgs::RemainingUsefulLife>("/remaining_useful_life", 1);
```

Comment:
These should actually be `/battery_state_of_charge` and `/battery_remaining_useful_life`, per the Google doc.  I haven't yet tested and will resume the review after this is corrected in this package and ow_plexil.